deploy/012-transfer-pools-ownership.ts

```diff
@@ -40,7 +17,7 @@ const transfer2StepOwnerships = async (contractNames: string[], targetOwner: str
     const pendingOwner = await contract.pendingOwner();
 
     let tx;
-    if (owner !== targetOwner && pendingOwner !== targetOwner) {
+    if (owner.toLowerCase() !== targetOwner.toLowerCase() && pendingOwner.toLowerCase() !== targetOwner.toLowerCase()) {
```

Comment:
I believe the convention should be to use checksummed addresses everywhere. This is what ethers returns in the calls, so we could avoid case conversions in this case.

Reply:
Updated